HW_1/task2.py

```diff
@@ -0,0 +1,16 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+s = int (input("Введите число "))
+rez = s % 6
+if rez == 0:
```

Comment:
Эта переменная используется единожды, всего лишь для одной проверки. Поэтому создавать ее нет никакой необходимости.
```suggestion
if s % 6 == 0:
```